ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/util/DbDispatcher.scala

```diff
@@ -65,8 +68,9 @@ private class DbDispatcherImpl(
           logger.error(s"got an uncaught exception on thread: ${thread.getName}", t))
         .build()))
 
-  override def executeSql[T](description: => String)(sql: Connection => T): Future[T] =
-    sqlExecutor.runQuery(description, () => connectionProvider.runSQL(conn => sql(conn)))
+  override def executeSql[T](description: String, extraLog: Option[String] = None)(
+      sql: Connection => T): Future[T] =
+    sqlExecutor.runQuery(description)(connectionProvider.runSQL(conn => sql(conn)))
```

Comment:
`extraLog` needs to be passed to `runQuery`.

Reply:
Gah. Thanks!

Reply:
Addresse by https://github.com/digital-asset/daml/pull/3310/commits/da69019b20e9b758d06008c4b2a5ab303bd1e4b6